doc/api/child_process.md

```diff
@@ -868,8 +868,10 @@ changes:
   * `output` {Array} Array of results from stdio output.
   * `stdout` {Buffer|string} The contents of `output[1]`.
   * `stderr` {Buffer|string} The contents of `output[2]`.
-  * `status` {number} The exit code of the child process.
-  * `signal` {string} The signal used to kill the child process.
+  * `status` {number|null} The exit code of the subprocess, or `null` if the
+    subprocess ended due to a signal.
```

Comment:
I don't object to "ended", but please be aware that its non-standard terminology: streams _end_, processes _terminate_, on both posixy and windowsy operating systems, see:
- https://docs.microsoft.com/en-us/windows/desktop/procthread/terminating-a-process
- https://linux.die.net/man/3/waitpid

Reply:
I'll switch to _terminate_ since it's the best term for this.